core/domain/classifier_services_test.py

```diff
@@ -103,7 +103,9 @@ def test_creation_of_jobs_and_mappings(self):
         # Modify such that job creation is triggered.
         new_answer_group = copy.deepcopy(state.interaction.answer_groups[1])
         new_answer_group.outcome.feedback.content_id = 'new_feedback'
+        new_answer_group.rule_specs[0].inputs['x']['content_id'] = 'new_ri'
```

Comment:
Is this what the content id is typically going to look like? Better to use one throughout that mimics the actual policy for setting such ids, to avoid confusion.